aten/src/TH/generic/THStorage.cpp

```diff
@@ -50,10 +44,8 @@ THStorage* THStorage_(newWithAllocator)(ptrdiff_t size,
   THStorage* storage = c10::make_intrusive<at::StorageImpl>(
```

Comment:
Need to verify call sites eventually set dtype

Reply:
Call sites look correct to me.